### PR TITLE
frontend: created and used MessageWaitDialog for send screen

### PR DESCRIPTION
**Pull request by shonsirsha**
Created and used MessageWaitDialog for send screen as a part of the `send.tsx` refactoring effort (#2190).

### CODE REVIEW

File: frontends/web/src/routes/account/send/send-guide.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ type TProps = {
     coinCode: CoinCode
 }
 
-export default function SendGuide({ coinCode }: TProps) {
```
Comment (shonsirsha) [inline frontends/web/src/routes/account/send/send-guide.tsx:11]: Using named export now. ✌️

File: frontends/web/src/routes/account/send/send-guide.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ type TProps = {
...
+export const SendGuide = ({ coinCode }: TProps) => {
```
Comment (shonsirsha) [inline frontends/web/src/routes/account/send/send-guide.tsx:11]: Using named export now. ✌️

File: frontends/web/src/routes/account/send/components/dialogs/message-wait-dialog.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  return (
+    <WaitDialog>
+      <div className="flex flex-row flex-center flex-items-center">
+        {icons[messageType]}
+        {messages[messageType]}
+      </div>
+    </WaitDialog>
+  );
```
Comment (thisconnect) [inline frontends/web/src/routes/account/send/components/dialogs/message-wait-dialog.tsx:33]: `switch(messageType)` would be nice

**Review by thisconnect — CHANGES_REQUESTED** [review]

File: frontends/web/src/routes/account/send/components/dialogs/message-wait-dialog.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <WaitDialog>
+      <div className="flex flex-row flex-center flex-items-center">
+        <IconAndMessage messageType={messageType} />
+      </div>
+    </WaitDialog>
+  );
+};
+
+const IconAndMessage = ({ messageType }: TIconProps) => {
```
Comment (shonsirsha) [inline frontends/web/src/routes/account/send/components/dialogs/message-wait-dialog.tsx:28]: Created a component that switches, PTAL @thisconnect  🙏

**Review by thisconnect — APPROVED** [review]
LGTM

**Pull request merged**